### ISSUE TITLE
Support for Cucumber 4+ message format

### ISSUE THREAD

**Issue by jmoglesby** [rajatthareja/ReportBuilder#93]
Cucumber 4+ has a default output format called [messages](https://github.com/cucumber/cucumber/tree/master/messages).

While I've been able to use the [stand-alone json formatter](https://github.com/cucumber/cucumber/tree/master/json-formatter) to convert `messages` into `json` for use with `report_builder`, it would be nice if `report_builder` supported the new native `messages` format.

Support for direct use of the `messages` format would be beneficial because:
1. I wouldn't have to maintain yet another library (`cucumber-json-formatter`) in my test environments
2. I could remove the extra processing step of converting from `messages` > `json` in my test scripts -- _any_ time saved in large regression runs is a worthwhile thing

Thank you!

**Comment by ChrisVereb** [comment]
Agreed!  There's a bugs in the standalone json formatter that is preventing it from  processing our results in 5.x and we've lost our reporting as a result.  While forking or updating this project to support the new message format may be  a large effort, it's something cucumber is definitely standardizing on going forward.  
A stopgap for many may be the integration of the standalone tool execution step into the report builder.   It would at least keep the project still technically compatible with the two most recent major versions of Cucumber.
